### PR TITLE
Update key-terms.md - changes to closed functionality key term examples

### PR DESCRIPTION
**Pull request by pday1**
Changes to the key term definition for closed functionality as per results for questions 4 & 5 from survey  https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-closed-more-to-review/results

### CODE REVIEW

File: key-terms.md [hunk 1]
```diff
@@ -21,19 +21,21 @@ The term **closed functionality**, as used in WCAG2ICT, has the meaning below:
 
 <DL><DT>closed functionality (as used in WCAG2ICT)</DT><DD>
   
-functionality that prevents users from attaching, installing, or using [assistive technology](#dfn-assistive-technology)</DD></DL>
+a property or characteristic that prevents users from attaching, installing, or using [assistive technology](#dfn-assistive-technology)</DD></DL>
```
Comment (maryjom) [inline key-terms.md:24]: I like this change!

File: key-terms.md [hunk 2]
```diff
@@ -21,19 +21,21 @@ The term **closed functionality**, as used in WCAG2ICT, has the meaning below:
...
+
+* self-service transaction machines or kiosks - examples include machines used for retail self-checkout, point of sales (POS) terminals, ticketing and self check-in, and Automated Teller Machines (ATMs).
+* telephony devices such as IP phones, feature phones, smartphones, and phone-enabled tablets (although such devices are becoming increasingly open and/or supplying AT and accessibility features that software can rely upon to meet WCAG requirements.)
+* entertainment technologies including smart TVs, set-top boxes, smart displays, smart speakers, smart watches, and tablets
+* an ebook reader or standalone ebook software that allows assistive technologies to access all of the user interface controls of the ebook program (open functionality) but does not allow the assistive technologies to access the actual content of book (closed functionality).
+* an operating system that requires the user to provide log in credentials before it allows any assistive technologies to be loaded. The log-in portion would be closed functionality.
+* other technology devices, such as printers, displays, and internet of things (iOt) devices.
```
Comment (maryjom) [inline key-terms.md:37]:
```suggestion
* other technology devices, such as printers, displays, and Internet of Things (IoT) devices. 
```

**Comment by GreggVan** [comment]
This line should be changed  — 

Closed functionality has to do with ability to work with AT  - not the ability to adjust things. 

> On Jan 24, 2024, at 8:50 AM, Mary Jo Mueller ***@***.***> wrote:
> -* computer that is locked down so that users may not adjust settings due to a policy

Suggest changing it to 

> -* computer that is locked down so that users are not able to install their AT

**Comment by maryjom** [comment]
> This line should be changed — Closed functionality has to do with ability to work with AT - not the ability to adjust things.
> On Jan 24, 2024, at 8:50 AM, Mary Jo Mueller ***@***.***> wrote: -* computer that is locked down so that users may not adjust settings due to a policy
> Suggest changing it to
> […](#)
> -* computer that is locked down so that users are not able to install their AT

@GreggVan You are commenting on content that was deleted using this PR. If you read the built document you can see this bullet is no longer there.

**Comment by GreggVan** [comment]
Ah 
Thanks 
Misunderstood.  Hard to get context when on an airplane and missing all my screens
g

**Review by bruce-usab — APPROVED** [review]
Thanks Phil!

I must confess that I am as prone as anyone to say/write "closed products" when I really mean "products that are closed to AT" and this revised definition supports that sort of practical short hand phrasing.

**Pull request merged**